repos/system_upgrade/el7toel8/actors/satellite_upgrade_data_migration/actor.py

```diff
@@ -38,11 +38,10 @@ def process(self):
                 os.unlink(service)
 
         if facts.postgresql.local_postgresql and os.path.exists(POSTGRESQL_SCL_DATA_PATH):
-            # remove empty PostgreSQL data from the package
-            if os.path.exists(POSTGRESQL_DATA_PATH):
-                os.rmdir(POSTGRESQL_DATA_PATH)
+            # we can assume POSTGRESQL_DATA_PATH exists and is empty
             # move PostgreSQL data to the new home
-            shutil.move(POSTGRESQL_SCL_DATA_PATH, POSTGRESQL_DATA_PATH)
+            for item in glob.glob(os.path.join(POSTGRESQL_SCL_DATA_PATH, '*')):
```

Comment:
Does this also cover hidden files?

Reply:
I don't think it does, but I'd also not expect any hidden files in that directory.

Reply:
I thought I had seen some files there in the past, like shell history files but couldn't find them now on my existing install. Then it would also have been a directory higher so :+1: from me.

Reply:
The home directory of the `postgres` user is `/var/lib/pgsql`, so if at all, things would end up there, not in `data`